src/main/java/Main.java

```diff
@@ -0,0 +1,42 @@
+import db.Storage;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import models.FruitTransaction;
+import service.Parser;
+import service.Processor;
+import service.Reader;
+import service.Writer;
+import service.impl.ParserImpl;
+import service.impl.ProcessorImpl;
+import service.impl.ReaderImpl;
+import service.impl.WriterImpl;
+import service.operation.BalanceHandler;
+import service.operation.OperationHandler;
+import service.operation.PurchaseHandler;
+import service.operation.ReturnHandler;
+import service.operation.SupplyHandler;
+import strategy.Strategy;
+import strategy.StrategyImpl;
+
+public class Main {
+    private static final String toFileName = "src/main/resources/testRecord.csv";
+    private static final String fromFileName = "src/main/resources/testData.csv";
+
+    public static void main(String[] args) {
+        Map<FruitTransaction.Operation, OperationHandler> operationHandlerMap = new HashMap<>();

```

Comment:
this is just a minor comment, maybe it's better to create the map in another class?